products-service/handler.ts

```diff
@@ -2,6 +2,10 @@ import  { getProductById } from './functions/getProductById';
 import  { getProductsList } from './functions/getProductsList';
 import 'source-map-support/register';
 import { APIGatewayProxyHandler } from 'aws-lambda';
+import { createProduct } from './functions/createProduct';
 
-export const getProductsListHandler: APIGatewayProxyHandler = async (event, _context) => getProductsList();
-export const getProductByIdHandler: APIGatewayProxyHandler = async (event, _context) => getProductById(event.path);
+
+
+export const getProductsListHandler: APIGatewayProxyHandler = async (event, _context) => await getProductsList();
+export const getProductByIdHandler: APIGatewayProxyHandler = async (event, _context) => await getProductById(event.pathParameters.id);
+export const createProductHandler: APIGatewayProxyHandler = async (event,_context) => await createProduct(JSON.parse(event.body));
```

Comment:
Just a curiosity: why do you need these wrappers?

Reply:
I use such wrappers in order to make my functions use only clean data without any additional fields